util/stream_dataset.py

```diff
@@ -383,12 +385,22 @@ def _insert_data(dataset, param, data, provenance_metadata=None, request_id=None
         """
         dims = ['obs']
 
-        # IF dimensions are defined in preload, use those
-        # otherwise, create dimensions dynamically based on the
-        # shape of the data
+        # the preload defined parameter dimensions
+        param_dimensions = []
         if param.dimensions:
-            dims += [d.value for d in param.dimensions]
+            param_dimensions = [d.value for d in param.dimensions]
+
+        if '-obs' in param_dimensions:
+            # remove obs dimension from parameter's dimensions and data (13025 AC2)
+            param_dimensions.remove('-obs')
```

Comment:
Has this been tested with the NetCDF download? NetCDF uses `obs` as the default dimension.

Reply:
this was tested and developed primarily for NetCDF data 13025, example netcdf output can be found attached to the ticket.